### DISCUSSION TITLE
INDIA - how long to wait?

### DISCUSSION THREAD

**Discussion by mgks** [community/community#6815] [Other Feature Feedback, Questions, & Ideas]
It's been more than 2 years since GitHub Sponsors released and I've been on that waitlist for so long now, I now go back and put my email every 6 months (I know it's unnecessary but I'm heartbroken).

I read in some recent official blog that it's still coming for Indian Developers. Just hear me out, there are around [10 million Indian Devs](https://www.thehindubusinessline.com/info-tech/github-eyes-10-million-developers-on-its-platform-in-india/article34167164.ece) on GitHub. Largest growing open source community from any other country, but still we are possibly among the last ones to get this feature (as you can see the current [countries](https://github.com/sponsors) who are enabled).

I mean, just do it @github, so long now.  :expressionless:

**Comment by ayan-biswas0412** [comment]
We are waiting for soo long, It is almost 2 years since the Github Sponsors has been launched but still it is not available for Indians.
Please @github make this feature available for us.

**Comment by Harindulk** [comment]
🥲🥲

**Comment by TarunavBA** [comment]
Yes I have also applied and am waiting...

**Comment by TarunavBA** [comment]
Today I recieved a mail from GitHub sponsors but unfortunately it is not about the sponsors feature for me, but it said that they are rolling out new OSS contributors on a daily basis in INDIA. I am eagerly waiting for it.....

**Reply by ayan-biswas0412** [reply]
yah same for me 🥲

**Comment by thetutlage** [comment]
Same here, I am waiting on the Github sponsors to open up in India so that I can create packages and learning content just for the sponsors.

In the meantime, I am using Patreon https://patreon.com/adonisframework, but the experience for the sponsors is not ideal and hinders my growth since not everyone goes through the pain of making a Patreon account and then sponsoring.

Please enable it for India 🙏

**Comment by jlord** [comment]
Hiya! I'm super happy to say [we're live in India now](https://github.blog/2022-05-23-github-sponsors-launches-in-india/)! Please let us know if you've got feedback and if you need support GitHub support covers Sponsors questions ✨

**Reply by thetutlage** [reply]
Yayyyy. You guys have made my day 🎉

**Reply by mgks** [reply]
finally! thank you guys.

**Reply by vishnupeas** [reply]
Holyyy this is huge

**Reply by thispsj** [reply]
Superb !

**Reply by Ayanmullick** [reply]
Will you support India's native payment platform, UPI, in the future?


https://indiastack.org/payments.html
https://medium.com/wharton-fintech/your-guide-to-upi-the-worlds-most-advanced-payments-system-b4e0b372bf0b
https://www.npci.org.in/what-we-do/upi/product-overview

**Reply by cosmicqbit** [reply]
Thank you very much 🎉

**Reply by cosmicqbit** [reply]
> Will you support India's native payment platform, UPI, in the future?
> 
> https://indiastack.org/payments.html https://medium.com/wharton-fintech/your-guide-to-upi-the-worlds-most-advanced-payments-system-b4e0b372bf0b https://www.npci.org.in/what-we-do/upi/product-overview

Yep this will really take the Github Sponsors to the next level.

**Comment by DhanushNehru** [comment]
I am from INDIA & I am super happy that I have received the Sponsor button in my github profile a few weeks back

**Comment by itsmeshibintmz** [comment]
I have Github sponsors option available now. I applied at the beginning of this month and registered.
